docs/configuration_file.md

```diff
@@ -203,6 +203,25 @@ Notes:
 for sells, please [open an issue](https://github.com/eprbell/dali-rp2/issues).
 * The Coincheck REST API only supports BTC trades, withdrawals, and deposits and there are currently no plans to implement it.
 
+### Kucoin Futures Section (REST)
```

Comment:
This should be moved to line 157 (at the end of the REST section in alphabetical order).

Reply:
Also, add an entry to the TOC up top.